### ISSUE TITLE
افزودن پشتیبانی زبان راست به چپ به بستر بلاگ‌نویسی پلام (plume)

### ISSUE THREAD

**Issue by ahangarha** [persian-free-software-ideas/persian-free-software-ideas#23]
<div dir=rtl>

**مشکل چیست؟ لطفا توضیح بدید**
بستر فدریتد پلام (plume) متنون راست‌به‌چپ را درست نشان نمی‌دهد. تصویر زیر، این موضوع را به خوبی نمایش می‌دهد:

![تصویر یک مطلب آزمایشی فارسی روی fediverse.blog که از پلام استفاده می‌کند](https://user-images.githubusercontent.com/11241315/57583213-7290c800-74e3-11e9-8e0e-fadbaadc6d7b.png)

**راه حلی را که دوست دارید ببینید بیان کنید**
احتمالا افزودن قابلیت پشتیبانی ساده باشد. باید هر بند یا پاراگراف خاصیت dir=rtl داشته باشد. به نظر نمی‌رسد حل این مشکل، دشوار باشد

**Comment by ahangarha** [comment]
<div dir="auto">
یک [issue](https://github.com/Plume-org/Plume/issues/575) مخزن اصلی باز کردم

**Comment by fdb-hiroshima** [comment]
(responding in english because I don't speak Persian). I think we will actually use `dir=auto`, so to delegate the job of figuring out direction to the browser, which will be easier for us. Unless there are reasons not to trust browser detection?

**Comment by ahangarha** [comment]
Yes. I think the solution is to add `dir` attribute to the elements. I trying to do it.
